### DISCUSSION TITLE
How can I get Moon v9 to have bicubic interpolation?

### DISCUSSION THREAD

**Discussion by VoidCretin** [d2phap/ImageGlass#1457] [Q&A] [answered]
On Kobe 8.7, I can change the image interpolation method from nearest neighbor, I was wondering if it was possible and if so how I would go about doing that.

**Comment by d2phap** [comment, marked as answer]
ImageGlass v9.0 beta 2 already supports it but **these settings are not exposed to the `igconfig.json` file**.
In beta 3 release, you will be able to change it by 2 settings:

```fs
{
  // when zoom is <= 100%
  "ImageInterpolationScaleDown": "SampleLinear",

  // when zoom is > 100%
  "ImageInterpolationScaleUp": "NearestNeighbor",
}
```

The supported image interpolation:
- `NearestNeighbor`
- `Linear`
- `Cubic`
- `SampleLinear`
- `Antisotropic`
- `HighQualityBicubic`

**Reply by VoidCretin** [reply]
Ok, thanks for clarifying!
